assets/js/script.js

```diff
@@ -258,9 +281,12 @@ function render() {
       var tab = Tab({ letter: l, material: state.materials[i], index: i })
       $('.js-loading').show()
       $('.js-letters').append(letter)
-      const tabSelector = ['.js-tab[data-index=',i-1,']'].join('')
+      const tabSelector = ['.js-tab[data-index=', i - 1, ']'].join('')
       $(tabSelector).parents('.js-tab-back').after(tab)
+
       letter[0].onload = function() {
+
+        $(this).css({ marginLeft: $(this).width() * letterSpacings[l.toUpperCase()]["left"] + "px", marginRight: $(this).width() * letterSpacings[l.toUpperCase()]["right"] + "px" })
```

Comment:
not sure how `this` would be the right thing here. I think it should be `letter.css`...